screen-home/src/main/java/com/ivy/home/HomeTab.kt

```diff
@@ -112,7 +112,8 @@ private fun BoxWithConstraintsScope.UI(
                 }
             )
     ) {
-        val listState = rememberLazyListState(
+        val listState = rememberScrollPositionListState(
+            key = "home_lazy_column",
```

Comment:
`rememberScrollPositionListState` must be applied to `:screen-transactions` too

Reply:
Done